java/vector/src/main/java/org/apache/arrow/vector/ipc/message/ArrowRecordBatch.java

```diff
@@ -79,14 +79,17 @@ public ArrowRecordBatch(int length, List<ArrowFieldNode> nodes, List<ArrowBuf> b
       arrowBuffers.add(new ArrowBuffer(offset, size));
       LOGGER.debug("Buffer in RecordBatch at {}, length: {}", offset, size);
       offset += size;
-      if (alignBuffers && offset % 8 != 0) { // align on 8 byte boundaries
-        offset += 8 - (offset % 8);
+      if (alignBuffers) { // align on 8 byte boundaries
+        offset = (offset + 7) & 0xfffffffffffffff8L;
```

Comment:
there are a few places where this code or similar occurs, can we find a centralized place to put it (same with code below).

Reply:
Specifically:
vector/src//main/java/org/apache/arrow/vector/BaseValueVector.java:    return (valueCount + 7) >> 3;
vector/src//main/java/org/apache/arrow/vector/BaseValueVector.java:    return ((size + 7) / 8) * 8;
vector/src//main/java/org/apache/arrow/vector/BitVectorHelper.java:    return (valueCount + 7) >> 3;

Reply:
Good suggestion. I have opened a separate class to place such utilities.